File: Groove.SP/groove-sp-web/src/app/core/models/shipments/shipment-bill-of-lading.model.ts
import { BillOfLadingModel } from 'src/app/features/shipment/bill-of-lading/models/bill-of-lading.model';
import { ModelBase } from '../model-base.model';

export interface ShipmentBillOfLadingModel extends ModelBase {
    shipmentId: number;
    billOfLadingId: number;
    billOfLading: BillOfLadingModel;
}

File: Groove.SP/Groove.SP.Core/Entities/ShipmentBillOfLadingModel.cs
using System;

namespace Groove.SP.Core.Entities
{
    public class ShipmentBillOfLadingModel : Entity
    {
        public long ShipmentId { get; set; }

        public long BillOfLadingId { get; set; }



        public virtual ShipmentModel Shipment { get; set; }

        public virtual BillOfLadingModel BillOfLading { get; set; }
    }
}
